assets/src/edit-story/elements/image/scalePanel.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import InOverlay from '../../components/overlay';
+
+const MIN_WIDTH = 165;
+const HEIGHT = 28;
+const OFFSET_Y = 8;
+// @todo: Should maxScale depend on the maximum resolution? Or should that
+// be left up to the helper errors? Both? In either case there'd be maximum
+// bounding scale.
+const MAX_SCALE = 400;
+
+const Container = styled.div`
+  position: absolute;
+  left: ${ ( { x, width } ) => `${ x + ( ( width - Math.max( width, MIN_WIDTH ) ) / 2 ) }px` };
+  top: ${ ( { y, height } ) => `${ y + height + OFFSET_Y }px` };
+  width: ${ ( { width } ) => `${ Math.max( width, MIN_WIDTH ) }px` };
+  height: ${ HEIGHT }px;
+
+  background: ${ ( { theme } ) => theme.colors.bg.v6 };
+  border-radius: 4px;
+
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  padding: 0 4px;
+`;
+
+const Range = styled.input`
```

Comment:
Nitpick: Maybe move non-dynamic attributes to `.attrs()` here (and remove them from the JSX):

```suggestion
const Range = styled.input.attrs( {
	type: "range",
	min: 100,
	max: MAX_SCALE,
	step: 10,
} )`
```

Reply:
Makes sense. I applied this suggestion. There might be a time when we may calculate the max scale, but for now this is better.